ocaml/xapi/repository.ml

```diff
@@ -304,6 +334,17 @@ let set_available_updates ~__context =
         Helpers.run_in_parallel funs capacity_in_parallel
     )
   in
+  let applied_livepatches_of_hosts =
```

Comment:
How many lines is this function long? I would encourage to split out more functions that are defined in this function.  If this is done just to keep the name space clean, I would use a local module rather than making this function larger and larger.

Reply:
Thanks. I split some functions out. Indeed, the `set_available_updates` function was too long.